specification/streamanalytics/resource-manager/readme.md

```diff
@@ -4,33 +4,75 @@
 
 This is the AutoRest configuration file for StreamAnalytics.
 
-
-
 ---
+
 ## Getting Started
+
 To build the SDK for StreamAnalytics, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the StreamAnalytics API.
 
 ``` yaml
 title: Stream Analytics Management Client
 description: Stream Analytics Client
 openapi-type: arm
-tag: package-2016-03
+tag: package-composite-v1
 ```
 
+### Tag: package-multi-api-version
```

Comment:
Also update the title?

Reply:
It should be. Thank you for catching it.